### PR TITLE
For next

### PR DESCRIPTION
**Pull request by hclee**

### CODE REVIEW

File: smb2pdu.c [hunk 1]
```diff
@@ -4138,12 +4139,6 @@ static void get_internal_info_pipe(struct smb2_query_info_rsp *rsp,
 	inc_rfc1001_len(rsp, sizeof(struct smb2_file_internal_info));
 }
 
-/**
- * smb2_info_file_pipe() - handler for smb2 query info on IPC pipe
- * @work:	smb work containing query info command buffer
```
Comment (namjaejeon) [inline smb2pdu.c:4143]: I thought you add correct arguments instead of removing comments...

Comment (hclee) [reply smb2pdu.c:4143]: I fixed this and other comments.

File: smb2pdu.c [hunk 2]
```diff
@@ -4815,12 +4817,6 @@ static int find_file_posix_info(struct smb2_query_info_rsp *rsp,
 	return 0;
 }
 
-/**
- * smb2_get_info_file() - handler for smb2 query info command
- * @work:	smb work containing query info request buffer
- *
```
Comment (namjaejeon) [inline smb2pdu.c:4821]: maybe, we can save this.

Comment (hclee) [reply smb2pdu.c:4821]: I added this comment.

**Pull request closed without merging**